nvflare/apis/fl_exception.py

```diff
@@ -25,4 +25,15 @@ def __init__(self, exception, message):
         super().__init__()
         # Copy all the gRPC exception properties into FLCommunicationError instance.
         self.__dict__.update(exception.__dict__)
-        self.message = message
+
+
+class WorkflowError(Exception):
```

Comment:
I think this can be simplified to "pass"...

Reply:
Removed no used WorkflowError